rllib/core/rl_module/rl_module.py

```diff
@@ -609,3 +609,7 @@ def as_multi_agent(self) -> "MultiAgentRLModule":
         marl_module = MultiAgentRLModule()
         marl_module.add_module(DEFAULT_POLICY_ID, self)
         return marl_module
+
+    def unwrapped(self) -> "RLModule":
+        """Returns the underlying module if this module is a wrapper."""
```

Comment:
Can you specify, what wrapper here means?
Like what are examples for RLModule wrappers?

Reply:
torch rl modules get wrapped with the torch ddp rl module wrapper

Reply:
done